test/docker-osx-dev.bats

```diff
@@ -415,3 +416,69 @@ export NEW_ENV_VARIABLE_2=VALUE2"
   run find_path_to_sync_parent "/some/path/.git/foo"
   assert_output '/some/path'
 }
+
+@test "get_json should get value for a single key json" {
+  output=$(echo -e '{ "foo": "bar" }' | get_json_value "foo")
+
+  assert_output "bar"
+}
+
+@test "get_json should get value for a nested json" {
+  output=$(echo -e '{ "foo": { "bar": "baz" } }' | get_json_value "bar")
+
+  assert_output "baz"
+}
+
+@test "get_json should get value for a json for a key that appears twice" {
+  output=$(echo -e '{ "foo": { "k": "first" },\n "k": "second" }' | get_json_value "k")
+
+  assert_output "first"
+}
+
+@test "get_json should not get value for a json for a non existent key" {
```

Comment:
What is the expected behavior of `get_json` if the key does exist, but its value is not a string, but an integer, object, or array?